apiary.apib

```diff
@@ -2469,15 +2470,611 @@ included in the output of this command.
 
 # Group dht
 
-## findpeer
+Issue commands directly through the DHT.
 
-## findprovs
+This command can't be called directly.
```

Comment:
I mean you can't call ` curl -i "http://localhost:5001/api/v0/dht"`. It makes sense with context, and I've used this notation elsewhere. I can change it to be more blatant?